app/model/sdl/Abstract/Model.js

```diff
@@ -200,48 +200,30 @@ SDL.SDLModel = Em.Object.extend({
   onFileRemoved: function(params) {
 
     var result = false;
+    var app_model = SDL.SDLController.getApplicationModel(params.appID);
 
     if ((
       params.fileType === 'GRAPHIC_PNG' || params.fileType === 'GRAPHIC_BMP' ||
```

Comment:
@AKalinich-Luxoft , I would advise to place condition `(params.fileType === 'GRAPHIC_PNG' || params.fileType === 'GRAPHIC_BMP' || params.fileType === 'GRAPHIC_JPEG')` into a single variable...

Or even make some lambda for that purpose, e.g:
```suggestion
      var file_is_pic = function (fileType) {
        var check_list = [ 'GRAPHIC_PNG', 'GRAPHIC_BMP', 'GRAPHIC_JPEG' ]
        for (var item in check_list) {
          if (fileType === item) {
            return true;
          }
        }
        return false;
      }
```

Reply:
@YarikMamykin good idea. However lambda is overhead here.
See https://github.com/smartdevicelink/sdl_hmi/pull/332/commits/ea6d7cf4860d59ef8e18d78749a13ae26bdaa0f3